plugins/vpc-shared-eni/network/bridge_linux.go

```diff
@@ -546,6 +546,13 @@ func (nb *BridgeBuilder) createBridge(
 				log.Errorf("Failed to enable IPv4 forwarding on %s: %v.", sharedENI.GetLinkName(), err)
 				return 0, err
 			}
+
+			// Set IPv4 proxy delay to 0 to avoid ARP proxy delays
```

Comment:
Can we please move this block to line 534? We should group ARP related settings together. This block should be right after the SetIPv4ProxyARP block. Please also update the comment to "Set IPv4 ARP proxy delay to zero to avoid delays.", with an "." at the end.